### PR TITLE
uniform tracing header format

### PR DESCRIPTION
**Pull request by sniperking1234**
uniform tracing header format

### CODE REVIEW

File: content/en/faq/distributed-tracing/how-to-support-tracing.md [hunk 1]
```diff
@@ -19,4 +19,4 @@ If you are using LightStep, you will also need to forward the following headers:
 
 - `x-ot-span-context`
 
-Header propagation may be accomplished through client libraries, such as [Zipkin](https://zipkin.io/pages/tracers_instrumentation.html) or [Jaeger](https://github.com/jaegertracing/jaeger-client-java/tree/master/jaeger-core#b3-propagation). It may also be accomplished manually, as documented in the [Distributed Tracing Task](/docs/tasks/observability/distributed-tracing/overview#trace-context-propagation).
\ No newline at end of file
+Header propagation may be accomplished through client libraries, such as [Zipkin](https://zipkin.io/pages/tracers_instrumentation.html) or [Jaeger](https://github.com/jaegertracing/jaeger-client-java/tree/master/jaeger-core#b3-propagation). It may also be accomplished manually, as documented in the [Distributed Tracing Task](/docs/tasks/observability/distributed-tracing/overview#trace-context-propagation).
```
Comment (frankbu) [inline content/en/faq/distributed-tracing/how-to-support-tracing.md:22]: This is strange, I don't see anything changed on this line. Why is it showing a diff?

File: content/zh/faq/distributed-tracing/how-to-support-tracing.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ Istio 允许报告服务网格中工作负载到工作负载间通信的追踪 s
 
 - `x-request-id`
 - `x-b3-traceid`
-- `x-b3-spanId`
+- `x-b3-spanid`
```
Comment (frankbu) [inline content/zh/faq/distributed-tracing/how-to-support-tracing.md:12]: It is recommended not to make changes to zh files in the same PR as en ones. The translation team will sync the zh file with the en change in a separate PR, after the en change has merged.

File: content/en/faq/distributed-tracing/how-to-support-tracing.md (hunk 1 above)
Comment (sniperking1234) [reply content/en/faq/distributed-tracing/how-to-support-tracing.md:22]: Probably because of the newline character.

**Pull request merged**